framework/packages/abstract-interface/Cargo.toml

```diff
@@ -12,7 +12,7 @@ daemon = ["cw-orch/daemon"]
 integration = []
 interchain = ["dep:cw-orch-interchain", "dep:cw-orch-polytone"]
 
-[dependencies]
```

Comment:
won't this fail to compile without an architecture flag in the library as well? (in lib.rs)

Reply:
> won't this fail to compile without an architecture flag in the library as well? (in lib.rs)

We already have it!
https://github.com/AbstractSDK/abstract/blob/e5f742d505b3934b2c83223b7464980b07fff47d/framework/packages/abstract-interface/src/lib.rs#L1